### PR TITLE
Testing api

### PR DESCRIPTION
**Pull request by Maicol-Avila**
what your name

### CODE REVIEW

File: src/app/layout.tsx
Comment (Bree2003) [inline src/app/layout.tsx:1]: quitar classname container

**Review by Bree2003 — CHANGES_REQUESTED** [review]
agregar classname container

**Review by PabloEmv — APPROVED** [review]
todo ok

**Review by ruribegSMU — CHANGES_REQUESTED** [review]
this can be changed

**Review by ruribegSMU — APPROVED** [review]
all ok

**Pull request merged**